### DISCUSSION TITLE
Import/Export snippets

### DISCUSSION THREAD

**Discussion by fredatgithub** [massCodeIO/massCode#266] [Q&A]
I have several computers with Masscode and I would like to export all my snippets from one computer and import them to another computer. So it would be great to add an export/import snippets feature.

**Comment by antonreshetov** [comment]
- First, you can use [sync](https://masscode.io/documentation/sync.html). 
- Second, if you want to manually transfer the snippets to another computer, just copy the database file and then [import](https://masscode.io/documentation/storage.html#importing) it into the application on the other computer.
